Formula/libepoxy.rb

```diff
@@ -11,8 +11,6 @@ class Libepoxy < Formula
     sha256 "f95aff4f5d3aed6991335ea6f67e3377a8a99200a554ca2cb0026c5e20630523" => :yosemite
   end
 
-  option :universal
```

Comment:
` ENV.universal_binary if build.universal?` still needs to be removed from this one.